lib/testing_api_onlyoffice_com/test_instance/main_page/document_builder_api/documentation_method_page.rb

```diff
@@ -0,0 +1,31 @@
+require 'onlyoffice_file_helper'
+module TestingApiOnlyfficeCom
+  # https://user-images.githubusercontent.com/40513035/57924882-0e894d80-78af-11e9-9ce3-d3f5eb9a2b23.png
+  # /docbuilder/basic
+  class DocumentationMethodPage
+    attr_accessor :link, :page
+
+    def initialize(editor, current_class, method)
+      @link = "#{TestingApiOnlyfficeCom::Config.new.server}/docbuilder/#{editor.downcase.tr_s(' ', '')}/#{current_class.downcase.tr_s(' ', '')}/#{method.downcase.tr_s(' ', '')}"
+      @page = Nokogiri::HTML(URI.parse(@link).open)
+    end
+
+    def all_elements_exist?
+      @params_exist ||= !@page.xpath('//*[@class="table"]').empty?
```

Comment:
I think `@params_exist ||= !@page.xpath('//*[@class="table"]').empty?` and other should be checked in constructor,
so method `all_elements_exist?` consiss only from ` @params_exist && @return_exist && @example_exist && @document_exist`
And by that it will be no need to call it in `not_existed_elements`